app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<header class="bg-primary p-2">
+  <h1 class="position-absolute">
+    <%= link_to root_path do %>
+      <i class="bi bi-caret-left-fill m-2 fs-4 bg-light rounded p-2 text-dark"></i>
+    <% end %>
+  </h1>
+  <h1 class="text-center text-light"> Log in </h1>
+</header>
+
+<div class="container mt-5">
```

Comment:
- [ ] Based on project requirements, 
<img width="767" alt="Screenshot 2022-07-01 at 13 12 26" src="https://user-images.githubusercontent.com/77038610/176864511-0081c072-086d-494e-8343-48a47ebc2002.png">
But I noticed that your design is very different from the provided one. 
Kindly update the views to make them as similar to the provided design as possible: 

Your Log in  | Behance Log in 
------------- | -------------
<img width="328" alt="Screenshot 2022-07-01 at 13 14 06" src="https://user-images.githubusercontent.com/77038610/176864848-9fc1e9f2-9a90-4e16-a6e9-cad8b6bb4127.png"> | <img width="233" alt="Screenshot 2022-07-01 at 13 14 37" src="https://user-images.githubusercontent.com/77038610/176864958-23e70a9e-40dd-423a-b18c-e5d6928a2814.png">